examples/3.audio_analysis/audio_classification/audio_classification.ipynb

```diff
@@ -2,12 +2,12 @@
  "cells": [
```

Comment:
<p>Let's either remove this output or keep other similar outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>